fragment-metadata.html

```diff
@@ -0,0 +1,162 @@
+        <input is="auth-creator" type="hidden" deer-key="creator" />
+        <input type="hidden" custom-key="source">
+        <div class="row">
+            <label class="col-3 col-2-md text-right is-hidden">Label:</label>
+            <input type="text" deer-key="title" placeholder="Label" class="col-9 col-10-md is-hidden">
+            <label class="col-3 col-2-md text-right">Shelfmark &#10082;</label>
+            <input type="text" deer-key="identifier" placeholder="A Shelfmark is required" class="col-9 col-10-md required" required>
+            <!--             
+                <label class="col-3 col-2-md text-right">Depictions:</label>
+                <input type="number" deer-key-todo="depiction" placeholder="" class="col-9 col-4-md"> 
+            -->
+            <label class="col-3 col-2-md text-right">Text: &#10082;</label>
+            <textarea custom-text-key="text" rows="2" class="col-9 col-10-md required" placeholder="Text is required.  Highlight some text..."></textarea>
+            <label class="col-3 col-2-md text-right">Language:</label>
+            <select custom-text-key="language" title="select language" class="col-9 col-4-md">
+                <option value="la" selected>Latin</option>
+                <option value="de">German</option>
+                <option value="fr">French</option>
+                <option value="en">English</option>
+            </select>
+            <label title="Select the checkbox if the text contains markup tags like HTML or XML" class="col-12 col-6-md text-left">
+                The text contains <code>&lt; tags &gt;</code><input type="checkbox" custom-text-key="format" value="text/plain" />
+            </label>
+            <label class="col-3 col-2-md text-right">Gloss format:</label>
+            <select deer-key="_glossFormat" title="select gloss format" class="col-9 col-4-md">
+                <option value="none" selected>None</option>
+                <option value="marginal">Marginal</option>
+                <option value="interlinear">Interlinear</option>
+                <option value="appendix">Appendix</option>
+                <option value="other">Other</option>
+            </select>
+            <!-- Useful but not used yet.  It is never shown in the UI and stays 'is-hidden' -->
+            <label class="col-3 col-2-md text-right is-hidden" placeholder="Chosen Gloss Incipit">Chosen Gloss</label>
+            <textarea disabled title="Choose a Gloss to see the incipit here" placeholder="Choose a Gloss below..." class="chosenGloss col-9 col-10-md is-hidden"></textarea>
+        </div>
+        <input class="is-hidden button primary" type="submit" value="Create Witness Fragment" />
+        <input class="is-hidden button error deleteWitness" type="button" value="Delete Witness Fragment" />
+        <div class="row">
+            <witness-text-selector class="lineSelector entity-needed is-hidden col-7-md col-6-sm"> </witness-text-selector>
```

Comment:
This makes me nervous because in this context, you would only select a single selection for this fragment. This page is not part of the Manuscript-first workflow like *upload file* or *link TPEN project*.

Reply:
Except this page will always have a created fragment that has text already selected. A fragment created from ng.html New Gloss already has text. Only the other workflow needs the selector.